### PR TITLE
Added upf-input class to password input

### PR DESCRIPTION
**Pull request by OwenCoogan**
### What does this PR do?

Capture d’écran 2023-10-23 à 11 33 12
It adds a class to fix the background color on password input.

Related to: #
https://linear.app/upfluence/issue/ENG-1426/password-field-empty-state-is-super-confusing

### Good PR checklist
- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [ ] Added/updated tests
- [ ] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: addon/components/o-s-s/password-input.hbs [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-      placeholder={{this.placeholder}}
-      autocomplete="current-password"
-      {{on 'keyup' this.validateInput}}
-    />
+    <Input class="upf-input"
+           @value={{@value}}
+           @type={{this.visibility}}
+           placeholder={{this.placeholder}}
+           autocomplete="current-password"
+           {{on 'keyup' this.validateInput}} />
```
Comment (Miexil) [inline addon/components/o-s-s/password-input.hbs:8]: might as well fix the double quotes around keyup while we're here :)

File: addon/components/o-s-s/password-input.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { tracked } from '@glimmer/tracking';
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 
-const DEFAULT_PLACEHOLDER = '••••••••••••';
+const DEFAULT_PLACEHOLDER = 'Enter your password';
```
Comment (Miexil) [inline addon/components/o-s-s/password-input.ts:7]: Don't hardcode text, please define this in the `en-us.yaml` file

**Review by JulienVannier66 — APPROVED** [review]
Otherwise LGTM

File: translations/en-us.yaml [hunk 3]
```diff
@@ -57,3 +58,4 @@ oss-components:
       subtitle: Failed to copy to your clipboard. Please try again.
   access-panel:
     search_placeholder: Search...
+
```
Comment (Miexil) [inline translations/en-us.yaml:61]: missing newline ? or extra newline ? either way github is saying something is wrong

**Review by Miexil — APPROVED** [review]

**Pull request merged**